src/webui/btjson.cpp

```diff
@@ -225,32 +225,36 @@ static const char KEY_LOG_PEER_IP[] = "ip";
 static const char KEY_LOG_PEER_BLOCKED[] = "blocked";
 static const char KEY_LOG_PEER_REASON[] = "reason";
 
-QVariantMap getTranserInfoMap();
-QVariantMap toMap(BitTorrent::TorrentHandle *const torrent);
-void processMap(QVariantMap prevData, QVariantMap data, QVariantMap &syncData);
-void processHash(QVariantHash prevData, QVariantHash data, QVariantMap &syncData, QVariantList &removedItems);
-void processList(QVariantList prevData, QVariantList data, QVariantList &syncData, QVariantList &removedItems);
-QVariantMap generateSyncData(int acceptedResponseId, QVariantMap data, QVariantMap &lastAcceptedData,  QVariantMap &lastData);
-
-class QTorrentCompare
+static QVariantMap getTranserInfoMap();
+static QVariantMap toMap(BitTorrent::TorrentHandle *const torrent);
+static void processMap(const QVariantMap &prevData, const QVariantMap &data, QVariantMap &syncData);
+static void processHash(QVariantHash prevData, const QVariantHash &data, QVariantMap &syncData, QVariantList &removedItems);
+static void processList(QVariantList prevData, const QVariantList &data, QVariantList &syncData, QVariantList &removedItems);
+static QVariantMap generateSyncData(int acceptedResponseId, const QVariantMap &data, QVariantMap &lastAcceptedData, QVariantMap &lastData);
```

Comment:
>With declarations at top and definitions at end?

You don't even need to do that IMO. The current namespace ends where the definitions of the said functions begin. So just move the curly braces and indent.